src/main/resources/data/pieces/1a902141-d8c6-4cf3-8a19-193eca007478.json

```diff
@@ -5,8 +5,8 @@
   "format": "Physical",
```

Comment:
PO Line has `"orderFormat": "Electronic Resource"` so please change `format` to `Electronic`

Reply:
format updated